launcher/src/store/services.js

```diff
@@ -170,6 +177,13 @@ export const useServices = defineStore("services", {
               icon: "/img/icon/plugin-menu-icons/fee.png",
               pattern: "(--suggested-fee-recipient=)(.*)(\\n)",
             },
+            {
+              title: "Doppelganger",
+              type: "toggle",
+              changeValue: true,
+              icon: "/img/icon/plugin-menu-icons/doppelganger.png",
+              pattern: "(--enable-doppelganger=)(.*)(\\n)",
```

Comment:
seems you have used the old pattern here. correct pattern would be the same as on line 97. this counts also for patterns on line 282, 387 and 821

Reply:
ok, those are the entries from the other clients, so please forget it. sunday morning is probably the wrong time for @daverolo to do a review ;)